lib/screens/v2/widget_instance/reconstructed_alert.ex

```diff
@@ -167,6 +179,22 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlert do
     end
   end
 
+  # Given an entity and the directionality of the alert from the home stop,
+  # return a tuple with the affected direction_id and route id
+  defp get_direction_and_route_from_entity(%{direction_id: nil, route: route}, location)
+       when location == :downstream,
```

Comment:
This and the pattern under it might be a little more readable without the guard:

```suggestion
  defp get_direction_and_route_from_entity(%{direction_id: nil, route: route}, :downstream),
```

Reply:
oh duh, thanks!